recipes/cd3-boost-unit-definitions/all/conanfile.py

```diff
@@ -41,7 +41,7 @@ def layout(self):
         basic_layout(self)
 
     def requirements(self):
-        self.requires("boost/1.72.0", transitive_headers=True)
+        self.requires("boost/[>=1.69.0]", transitive_headers=True)
```

Comment:
Ranged version is not allowed in CCI, use only specific version please.

Reply:
Oops, sorry. So what is the recommendation for selecting the version? Should it be the newest version available at the time, or the oldest version supported?

Reply:
Newest available :)

Reply:
Todo for me but the docs are sucky on this one

https://github.com/conan-io/conan-center-index/blob/3b5742afddd89408c6343dd18b173283fbb84818/docs/faqs.md?plain=1#L355

https://github.com/search?q=repo%3Aconan-io%2Fconan-center-index%20version%20ranges&type=code

Reply:
You got it.